packages/react/src/components/Grid/Grid.js

```diff
@@ -51,6 +76,11 @@ Grid.propTypes = {
    */
   className: PropTypes.string,
 
+  /**
+   * Specify how many columns wide the Grid should span
+   */
+  columns: PropTypes.number,
```

Comment:
What do we need this `columns` prop for? 👀

Reply:
Provides control over how many columns are available in the grid in total. The most common would be overriding the default 16 to a 12 column grid for equal spacing of thirds. 

For a subgrid this controls how many columns it will span.

Reply:
> The most common would be overriding the default 16 to a 12 column grid for equal spacing of thirds.

This seems like it might be against IDL stuff, right? I thought for thirds they basically say drop the last column and do 5/5/5 but I could be misremembering 🤔 

> For a subgrid this controls how many columns it will span.

If I remember correctly this should be handled by the column classes in CSS, right? Is there somewhere in the React code where we need this?

Reply:
I've never heard of the 5/5/5! 😅  That could totally be my lack of knowledge on the topic. Is it on the IDL site or in our docs?

> If I remember correctly this should be handled by the column classes in CSS, right? Is there somewhere in the React code where we need this?

Yeah, this controls which column class is applied. By default it's 16, but some users may want a subgrid to be 8, to enable stuff like another non-subgrid on the same "row".

Reply:
Nice, makes sense!